<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DProbeAtomDispl</class>
 <widget class="QDialog" name="DProbeAtomDispl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Results of the calculation of the dislocation position</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>580</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpAtomProp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Save properties of atoms</string>
   </property>
   <widget class="QCheckBox" name="chkxyzInit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>coordinates of atoms in the initial configuration</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkAtomPosInit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>401</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>position of atom along the chosen path (initial config.)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkAtomNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>atom number</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkAtomPosRel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>411</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>position of atom along the chosen path (relaxed config.)</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkComment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>comment at the beginning starting with #</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="fileView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>961</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btSaveData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save data...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpAtomAux">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Save auxiliary fields</string>
   </property>
   <widget class="QCheckBox" name="chkScrew">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>cumulative differential displacements - screw comp.</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkEdge">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>cumulative differential displacements - edge comp.</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkRelDispZ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>relative displacemets of atoms in the Z direction</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>File view</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkxyzRel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>coordinates of atoms in the relaxed configuration</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DProbeAtomDispl</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DProbeAtomDispl</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
